src/compute/src/logging/compute.rs

```diff
@@ -139,7 +139,13 @@ pub fn construct<A: Allocate>(
             let mut peek_stash = BTreeMap::new();
             let mut dataflow_imports = BTreeMap::<
                 (GlobalId, usize),
-                BTreeMap<GlobalId, (VecDeque<(mz_repr::Timestamp, u128)>, BTreeMap<u128, i32>)>,
+                BTreeMap<
+                    GlobalId,
+                    (
+                        VecDeque<(mz_repr::Timestamp, u128)>,
+                        BTreeMap<u128, (i128, i32)>,
```

Comment:
Could you add a description on what the types mean? Maybe it's time to add a struct?

Reply:
Sure, I did both